csrc/kernel_cache.cpp

```diff
@@ -672,6 +672,12 @@ FusionKernelRuntime::FusionKernelRuntime(
   optimization::OptimizationPass<optimization::PreSegmenter>::runPass(
       fusion.get());
 
+  if (isDebugDumpEnabled(DebugDumpOption::FusionIrPreseg)) {
+    std::cout << "Fusion IR after pre-segmenter optimization passes:"
+              << std::endl;
+    fusion->printMath();
+  }
```

Comment:
New dump option `fusion_ir_preseg` to more easily monitor what the optimization passes are doing.